deployment/build_and_release/live/ci/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  backend "gcs" {}
+}
+
+module "shared" {
+  source = "../../../../../modules"
```

Comment:
Traversing backwards is a bit grim, which is why we use the `${get_path_to_repo_root()}` method. It's really hard to reason about the number of backsteps that are going on here.